gcp_variant_transforms/beam_io/vcfio.py

```diff
@@ -843,6 +846,19 @@ def _get_field_count_as_string(self, field_count):
       else:
         raise ValueError('Invalid value for field_count: %d' % field_count)
 
+    def _get_variant_end_from_info(self, end_info_value):
+      """Returns the value from the END INFO key."""
+      if isinstance(end_info_value, (int, long)):
+        return end_info_value
+      elif (isinstance(end_info_value, list) and len(end_info_value) == 1 and
+            isinstance(end_info_value[0], (int, long))):
+        # Some VCF files set the number of END info fields to unknown instead
+        # of one, which parses into a list in PyVCF.
+        return end_info_value[0]
+      else:
+        raise ValueError('Invalid END INFO field in record: {}'.format(
+            self._last_record))
+
```

Comment:
This doesn't need to be done now, but I think we should add a TODO about refactoring convert_to_variant_record. It should probably be broken into several methods like the `_get_variant_end_from_info` like you did here.